jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/memory/ObjectAllocationOutsideTLABHandler.java

```diff
@@ -59,19 +57,19 @@ public String getEventName() {
   private static class PerThreadObjectAllocationOutsideTLABHandler
       implements Consumer<RecordedEvent> {
     private static final String ALLOCATION_SIZE = "allocationSize";
-    private static final String MAIN = "Main";
 
-    private final BoundDoubleHistogram boundHistogram;
+    private final DoubleHistogram histogram;
+    private final Attributes attributes;
 
     public PerThreadObjectAllocationOutsideTLABHandler(
         DoubleHistogram histogram, String threadName) {
-      boundHistogram =
-          histogram.bind(Attributes.of(ATTR_THREAD_NAME, threadName, ATTR_ARENA_NAME, MAIN));
+      this.histogram = histogram;
+      this.attributes = Attributes.of(ATTR_THREAD_NAME, threadName, ATTR_ARENA_NAME, "Main");
```

Comment:
Should probably dig into / have more discussion around Arena naming - I am not 100% that my understanding of what JFR presents to us is correct.